teams/kubernetes/README.md

```diff
@@ -0,0 +1,16 @@
+# Deployment Team
+
+This team works for Kubernetes and Docker integration.
+
+* Supporting new features or improving performance for TiDB Products on Kubernetes and Docker.
+* Supporting TiDB ecosystem tools integrated natively with TiDB on Kubernetes.
+
+## Members
+
+See [membership.json](membership.json)
+
+## Repositories
+
+* [tidb-operator](https://github.com/pingcap/tidb-operator)
+* [tidb-docker-compose](https://github.com/pingcap/tidb-docker-compose)
```

Comment:
This should have been deprecated, right?
It's not been maintained for a long time.

Reply:
https://github.com/pingcap/tidb-docker-compose/pull/96#issuecomment-866683551 from this comment I think @alex-quan-001 willing to take over the project. You may have a discussion to see where it to go. We can always evict the project from kubernetes team for now and leave the question for later.

Reply:
> not been maintained for a long t

There was an issue recently opened regarding tidb-docker-compose component, and it seems not working somehow. Maybe archiving this repository can make it clear that people should looking into other options like TiUP.

Reply:
@DanielZhangQD I'm going to remove this repo from our config for now and postpone the solution after we have a discussion with @alex-quan-001.